docs/README.md

```diff
@@ -0,0 +1,39 @@
+# PyPowSyBl documentation
+
+These are the documentation sources for PyPowSyBl features.
+
+Please keep them up to date with your developments.  
+They are published on powsybl.readthedocs.io/ and pull requests are built and previewed automatically.
```

Comment:
Shouldn't we write on powsybl.readthedocs.io/projects/pypowsybl or on pypowsybl.readthedocs.io?